easy-ui-react/src/TextField/TextField.module.scss

```diff
@@ -1,6 +1,8 @@
 @use "../styles/common" as *;
 
 .root {
+  @include component-token("textfield", "min-height-textarea", 48px);
+  @include component-token("textfield", "min-height-textarea-lg", 56px);
```

Comment:
figma suggests min height of `32px` for the standard size, but i think its intended to be `48px`, will check/confirm though